src/main/java/org/sagebionetworks/web/client/widget/entity/EntityMetadata.java

```diff
@@ -188,11 +192,15 @@ public void onFailure(Throwable t) {
     );
   }
 
+  @Override
   public void setAnnotationsVisible(boolean visible) {
+    annotationsAreVisible = visible;
     if (DisplayUtils.isInTestWebsite(ginInjector.getCookieProvider())) {
-      view.setAnnotationsModalVisible(visible);
+      entityModalWidget.setOpen(visible);
+      view.setAnnotationsVisible(false);
     } else {
       view.setAnnotationsVisible(visible);
+      entityModalWidget.setOpen(false);
```

Comment:
In addition to showing the correct annotation viewer based on experimental mode, hide the other one